migrate/db-seed-commonjs-pkg/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "db-seed-commonjs-pkg",
+  "devDependencies": {
+    "@types/node": "12.20.23",
+    "prisma": "2.31.0-dev.46",
+    "ts-node": "10.2.1",
+    "jest": "26.6.3",
+    "prettier": "2.3.1"
+  },
+  "dependencies": {
+    "@prisma/client": "2.31.0-dev.46"
+  },
+  "scripts": {
+    "test": "jest --forceExit"
```

Why `forceExit`? I think we should Jest let us tell about stuff that stays open:
```suggestion
    "test": "jest"
```